ci_framework/roles/copy_container/tasks/main.yml

```diff
@@ -38,8 +38,9 @@
   register: go_build
   ci_script:
     output_dir: "{{ cifmw_artifacts_basedir }}/artifacts"
-    cmd: go build
-    chdir: "{{ temporary_copy_container_dir.path }}"
```

Comment:
ci_script should support the same parameters as the original ansible.builtin.script:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/script_module.html#parameter-chdir

there's something else then... Do you have some link?

Reply:
dang... ok, I'm wrong:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/plugins/action/ci_script.py#L131